free-programming-books-tr.md

```diff
@@ -124,6 +124,7 @@
 ### Python
 
 * [Python ile Programlama](https://belgeler.yazbel.com/python-istihza/)
+* [Bilgisayar Bilimcisi Gibi Düşünmek-Python3 ile Öğrenme](http://ofenerci.github.io/thinkcspy-tr)
```

Comment:
One of my favorite free-licensed programming resources, great to see a Turkish version!
1. need to put the lists in alphabetical order by title.
2. consider adding the original author after the link to help identify the work. see issue #2383 
3. take a look at issue #2357 about in-process works